src/io/dirops.c

```diff
@@ -13,55 +13,25 @@
 #  define PATH_MAX 2048
 #endif
 
-#ifdef _WIN32
-static wchar_t * UTF8ToUnicode(char *str)
-{
-     const int         len = MultiByteToWideChar(CP_UTF8, 0, str, -1, NULL, 0);
-     wchar_t * const result = (wchar_t *)MVM_malloc(len * sizeof(wchar_t));
-
-     MultiByteToWideChar(CP_UTF8, 0, str, -1, result, len);
-
-     return result;
-}
-
-static char * UnicodeToUTF8(const wchar_t *str)
-{
-     const int       len = WideCharToMultiByte(CP_UTF8, 0, str, -1, NULL, 0, NULL, NULL);
-     char * const result = (char *)MVM_malloc(len * sizeof(char));
-
-     WideCharToMultiByte(CP_UTF8, 0, str, -1, result, len, NULL, NULL);
-
-     return result;
-}
-
-static int mkdir_p(MVMThreadContext *tc, wchar_t *pathname, MVMint64 mode) {
-    wchar_t *p = pathname, ch;
-#else
 static int mkdir_p(MVMThreadContext *tc, char *pathname, MVMint64 mode) {
     char *p = pathname, ch;
     uv_fs_t req;
     int mkdir_error = 0;
-#endif
     int created = 0;
 
     for (;; ++p)
         if (!*p || IS_SLASH(*p)) {
             ch = *p;
             *p  = '\0';
-#ifdef _WIN32
-            created = CreateDirectoryW(pathname, NULL);
-#else
-            created = (mkdir(pathname, mode) == 0
-                       || ( (mkdir_error = errno) == EEXIST
-                       && uv_fs_stat(NULL, &req, pathname, NULL) == 0
-                       && S_ISDIR(req.statbuf.st_mode)));
-#endif
+            created = (mkdir_error = uv_fs_mkdir(NULL, &req, pathname, mode, NULL) == 0
```

Comment:
Does this need an extra set of parens? `(mkdir_error = uv_fs_mkdir(NULL, &req, pathname, mode, NULL)) == 0`